testkit/src/main/scala/com/gu/scanamo/LocalDynamoDB.scala

```diff
@@ -13,6 +14,7 @@ object LocalDynamoDB {
       .asyncBuilder()
       .withCredentials(new AWSStaticCredentialsProvider(new BasicAWSCredentials("dummy", "credentials")))
       .withEndpointConfiguration(new EndpointConfiguration("http://localhost:8042", ""))
+      .withClientConfiguration(new ClientConfiguration().withClientExecutionTimeout(50000).withRequestTimeout(5000))
```

Comment:
why do we need this new line?

Reply:
yes. it was always some alpakka test failing so i increased the patience config there, but this line above cannot hurt anyway.